src/app/hooks/staking/useStaking_timestampToLockDate.ts

```diff
@@ -0,0 +1,9 @@
+import { useCacheCallWithValue } from '../useCacheCallWithValue';
+
+export const useStaking_timestampToLockDate = (timestamp: number) =>
+  useCacheCallWithValue(
+    'staking',
+    'timestampToLockDate',
+    '0',
+    Number(timestamp),
```

Comment:
Why do you need to cast `timestamp`? It's already a `number`.

Reply:
Ops